src/sync.js

```diff
@@ -184,6 +184,11 @@ class Sync {
       case side == null:
         this.pouch.setLocalSeq(change.seq, callback)
         break
+      case (sideName === 'remote' && doc.toBeTrashed && !inRemoteTrash(doc)):
+        // File or folder was just deleted locally
+        this.trashLaterWithParentOrByItself(doc, side)
+        done()
```

Comment:
Makes sense.
I believe not calling `done()` immediately was prevent upcoming changes to pop up, making it impossible to "wait" for some trashed parent dir.
Not sure how to fix it yet.